embed-code-go/go.mod

```diff
@@ -21,9 +21,11 @@ module embed-code/embed-code-go
 go 1.22.1
 
 require (
+	github.com/bmatcuk/doublestar/v4 v4.6.1 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/gobwas/glob v0.2.3 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/stretchr/testify v1.9.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
If this is our dependency why can't we use `v3.0.1` the same as the `indirect` one? Because two versions of same library confuse me. Especially when the manually added one is not the latest one.